client/apps/edit/components/content2/sections/text/index.styl

```diff
@@ -84,5 +91,27 @@
 .feature .edit-section--text
   blockquote
     Unica s69
-
+  h1
+    Unica s40
+    text-align center
+    padding-bottom 46px
+    padding-top 107px
+    position relative
+    &:before
+      position absolute
+      top 69px
+      right calc(50% \- 7.5px)
+      content '.'
+      height 15px
+      width @height
+      background black
+      border-radius 50%
+      color rgba(0,0,0,0)
+  .content-start
+    Unica s69, medium
+    float left
+    line-height .5em
+    margin-right 10px
+    margin-top .298em
+    text-transform uppercase
```

Comment:
It would be nice to eventually wrap text in a Reaction component that contains all these styles.